cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -140,13 +141,27 @@ void AnimatedImageTracker::Reset() {
 }
 
 void AnimatedImageTracker::OnRenderTreeRasterized() {
```

Comment:
OOC, how often and when is this method called?

Reply:
Every rendered frame, which hopefully stays above 30fps.

Reply:
Definitely a histogram use case then.